app/Domain/Integrations/FormRequests/RequestActivationRequest.php

```diff
@@ -45,4 +52,12 @@ private function isAccountingInfoRequired(): bool
 
         return $integration->type !== IntegrationType::EntryApi || $subscription->price > 0.0;
     }
+
+    private function isUITPAS(): bool
+    {
+        /** @var IntegrationRepository $integrationRepository */
+        $integrationRepository = App::get(IntegrationRepository::class);
+        $integration = $integrationRepository->getById(Uuid::fromString($this->id));
```

Comment:
Would maybe extract fetching the Integration from the two boolean check methods and pass it as argument so that it isn't done twice.